qcodes_contrib_drivers/drivers/Keysight/Keysight_E8267D.py

```diff
@@ -142,10 +154,10 @@ def __init__(self, name, address, step_attenuator=False, **kwargs):
         self.connect_message()
 
     def on(self):
-        self.set('status', 'on')
+        self.set('output_rf', 'on')
 
     def off(self):
-        self.set('status', 'off')
+        self.set('output_rf', 'off')
```

Comment:
this should probably be improved further in order to get further away from string-specifying parameters:
```suggestion
        self.output_rf('off')
```